platform/src/main/java/org/hillview/dataset/api/IMonoid.java

```diff
@@ -32,8 +32,11 @@
 
     @Nullable
     default R reduce(List<R> data) {
-        // This implementation avoids allocating a zero
-        // if the list is non-empty.
+
+        /**
```

Comment:
Don't use JavaDoc comments for code, only for fields, classes, etc.

Reply:
Sure